### PR TITLE
Updating SDK to incorporate new one-of schema

### PR DESCRIPTION
**Pull request by ghost**
## Changes introduced with this PR

Updating SDK to incorporate arcalot/arcaflow-plugin-sdk-python#72
---
By contributing to this repository, I agree to the [contribution guidelines](https://github.com/arcalot/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 .idea
 *.iml
+__pycache__
```
Comment (sandrobonazzola) [inline .gitignore]: missing newline at eof

**Review by sandrobonazzola — COMMENTED** [review]
minor comment.

File: Dockerfile [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
-RUN dnf module -y install python39 && dnf install -y python39 python39-pip lksctp-tools-devel git
+RUN dnf module -y install python39 && dnf install --setopt=tsflags=nodocs -y python39 python39-pip lksctp-tools-devel && dnf clean all
 COPY --from=builder /usr/local/bin/uperf /usr/local/bin/uperf
 RUN mkdir /plugin
-ADD https://raw.githubusercontent.com/arcalot/arcaflow-plugins/main/LICENSE /plugin
-ADD uperf_plugin.py /plugin
-ADD uperf_schema.py /plugin
-ADD test_uperf_plugin.py /plugin
-ADD requirements.txt /plugin
+ADD https://raw.githubusercontent.com/arcalot/arcaflow-plugins/main/LICENSE /plugin/
```
Comment (portante) [inline Dockerfile:10]: Should we be using the `LICENSE` of this plugin?

File: Dockerfile [hunk 3]
```diff
@@ -1,20 +1,22 @@
...
+ADD uperf_plugin.py /plugin/
```
Comment (portante) [inline Dockerfile:11]: Should we be using `COPY` here instead?

**Review by portante — CHANGES_REQUESTED** [review]

**Comment by dustinblack** [comment]
@janosdebugs What do we need to do to get this one moved forward? I've been running quite a few successful workflow tests using the development build from this PR.

**Comment by ghost** [comment]
This can be merged from my pov, the docs will need to be updated later

**Comment by jaredoconnell** [comment]
This PR is stale.

**Pull request closed without merging**